### PR TITLE
Updated Buyer's Guide og tag title, description, and image

### LINKED ISSUE #2184: Update OG image and copy
#### Current OG copy and image:
image

#### Please update copy to:

Title: *Privacy Not Included: A Holiday Buyer’s Guide for Connected Products
Intro: 70 connected products rated on their safety and security

#### And update image to:

09-og

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2184

### CODE REVIEW

File: network-api/networkapi/buyersguide/templates/bg_base.html [hunk 1]
```diff
@@ -5,10 +5,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta property="og:type" content="website" />
     {% block og-title %}
-      <meta property="og:title" content="privacy not included" />
+      <meta property="og:title" content="*Privacy Not Included: A Holiday Buyer’s Guide for Connected Products" />
     {% endblock %}
-    <meta property="og:description" content="This year, learn what tech comes with privacy included, using Mozilla's buyer's guide for connected gadgets." />
+    <meta property="og:description" content="70 connected products rated on their safety and security" />
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/bg_base.html]: Should we make this dynamic? Like `content="{{ products.all | len }} connected products ..."`?

**Comment by mmmavis** [comment]
### do not review yet

image

Description is missing in the preview. One online post says the title needs to be under 63 characters long for the description to show on Facebook. Let me test & verify it.

**Comment by kristinashu** [comment]
Ah ok, shoot. Could probably just do `*Privacy Not Included: A Holiday Buyer’s Guide` for the title but will confirm with Jen tomorrow (she's done for the day).

**Comment by mmmavis** [comment]
@kristinashu No worries! Can you update the ticket (or here) and tag me when you get Jen's confirmation? Thanks!

**Comment by kristinashu** [comment]
Jen says to use this title: *Privacy Not Included: A Buyer’s Guide for Connected Products

**Comment by kristinashu** [comment]
So removing the word "holiday"

**Comment by mmmavis** [comment]
image

you can test it out here as well: https://developers.facebook.com/tools/debug/

**Comment by kristinashu** [comment]
That debugger gives this warning, does that matter?

image

**Comment by mmmavis** [comment]
Missing `fb:app_id` should be fine! Thanks for catching that though.

**Pull request merged**